api/simplex/tools/conversion_tools.py

```diff
@@ -0,0 +1,22 @@
+from api.simplex.sub_classes.target_class import TargetType
+from api.simplex.sub_classes.sign_class import Sign
+
+def get_target_from_string(string_target: str) -> TargetType or None:
+    match string_target:
+        case "Minimize":
+            return TargetType.MIN
+        case "Maximise":
+            return TargetType.MAX
+        case _:
+            return None
+
+def get_sign_from_string(string_sign: str) -> Sign or None:
+    match string_sign:
+        case ">=":
+            return Sign.GREATER_EQ
+        case "=":
+            return Sign.EQ
+        case "<=":
+            return Sign.SMALLER_EQ
+        case _:
+            return None
```

Comment:
Why put these helpers in the same file, but not the classes ?

I would probably add a static `from_string` method in the classes themselves

Reply:
Something like this:
```py
class Sign:
    GREATER_EQ = ">="
    SMALLER_EQ = "<="
    EQ = "="

    @staticmethod
    def from_string(sign_str: str) -> 'Sign':
        match sign_str:
            case ">=":
                return Sign.GREATER_EQ
            case "<=":
                return Sign.SMALLER_EQ
            case "=":
                return Sign.EQ
        raise ValueError(f"Invalid sign: {sign_str}")

print(Sign.from_string("<="))
```